src/sentry/shared_integrations/exceptions/base.py

```diff
@@ -64,7 +63,11 @@ def __str__(self) -> str:
 
     @classmethod
     def from_response(cls, response: Response, url: str | None = None) -> ApiError:
-        from sentry.shared_integrations.exceptions import ApiRateLimitedError, ApiUnauthorized
+        from sentry.shared_integrations.exceptions import (
+            ApiConflictError,
+            ApiRateLimitedError,
+            ApiUnauthorized,
+        )
```

Comment:
ideally we should fix the cycle instead of making it worse -- these two modules are tightly coupled so they should probably just be one module

Reply:
Good point, mind I do this in a follow-up PR?